src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2284,9 +2284,14 @@ protected virtual bool ShouldMarkInterfaceImplementation (TypeDefinition type, I
 			if (Annotations.IsMarked (iface))
 				return false;
 
-			if (Annotations.IsMarked (resolvedInterfaceType) && !Annotations.IsMarked (iface))
+			if (Annotations.IsMarked (resolvedInterfaceType))
```

Comment:
Where did `!Annotations.IsMarked (iface))` check go?

Reply:
Annotations.IsMarked (iface) is already false by the time we get here due to the check above.